code/test/core.js

```diff
@@ -103,9 +104,28 @@ contract('ONEWallet', (accounts) => {
   Logger.debug(`Testing with ${accounts.length} accounts`)
   Logger.debug(accounts)
   let snapshotId
+  let alice, bob, carol, dora, ernie, state, bobState, carolState, doraState, ernieState, testerc20, testerc721, testerc1155, testerc20v2, testerc721v2, testerc1155v2
+
   beforeEach(async function () {
     await TestUtil.init()
     snapshotId = await TestUtil.snapshot()
+    const testData = await TestUtil.deployTestData()
```

Comment:
could create one object then assign the properties to `global`

Reply:
refactored to use a similar approach to factories.
However could use an example of how to implement `global` 
Read [this](https://developer.mozilla.org/en-US/docs/Glossary/Global_object) but did not understand how to apply it in this case
*Note: Unlike [var](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var), the statements [let](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) and [const](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const) do not create properties of the global object.*
Tried using `var` but I think I'm missing something.